spec/models/user_spec.rb

```diff
@@ -32,5 +32,5 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  it { should have_many(:timeslots).with_foreign_key(:default_user_id) }
```

Comment:
Just for my learning purpose: 
You removed this line because at the User model you also removed 
`  has_many :timeslots, foreign_key: :default_user_id, inverse_of: :default_user,
                       dependent: :nullify` (lines 41-42)
If this line had not been removed, there will be some failed test cases because it expects the User to have many :timeslots. 

What would happen if we were just to delete this entire file altogether? It seems like currently this test doesn't do anything?

Reply:
After looking at #13, it seems that user_spec.rb should not be deleted because there could be other tests that we may want to run on the user model. Please correct me if I'm wrong

Reply:
I haven't deleted the file, just deleted this line as there are no more timeslots. 
If you delete the test, there will be no failures.
If you delete lines 41-42, then there might be failures.